uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/ScrollableTabRow.kt

```diff
@@ -0,0 +1,379 @@
+package io.github.droidkaigi.feeder.core
+
+/**
+ * Copied from [androidx.compose.material.ScrollableTabRow] to support background indicator.
+ */
```

Comment:
This change may make it difficult to keep up with library updates. 😔 
Is there a good way to customize the Composable function?

Reply:
I learned a lot from this PR. Thank you very much.
How about proxying with SubcomposeLayout?

```kotlin
fun ScrollableTabRowWithForeground() {
       val indicatorTabPosition by remember{ mutableStateOf() }
       val tabRowPlaceables = subcompose(SlotsEnum.Main) {
            ScrollableTabRow(
                ...
              indicator: @Composable (tabPositions: List<TabPosition>) -> Unit = @Composable { tabPositions ->
                  val indicatorPosition = Modifier.tabIndicatorOffset(tabPositions[selectedTabIndex])
                  TabRowDefaults.Indicator(
                       Modifier.tabIndicatorOffset(indicatorPosition)
                   )
                   indicatorTabPosition = indicatorPosition
              },
            )
        }.map {
            it.measure(constraints)
        }
        layout(...) {
            // layout indicator with indicatorPosition and tabRowPlaceables
        }
```
}

Reply:
Thanks for the suggestion! I'll try it ! 😃